line-bot-spring-boot/src/main/java/com/linecorp/bot/spring/boot/support/ReplyByReturnValueConsumer.java

```diff
@@ -0,0 +1,116 @@
+package com.linecorp.bot.spring.boot.support;
+
+import static java.util.Collections.singletonList;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Consumer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+
+import com.linecorp.bot.client.LineMessagingClient;
+import com.linecorp.bot.model.ReplyMessage;
+import com.linecorp.bot.model.event.Event;
+import com.linecorp.bot.model.event.Event.ReplySupport;
+import com.linecorp.bot.model.message.Message;
+import com.linecorp.bot.model.response.BotApiResponse;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Internal class to send message as reply.
+ *
+ * Basically, message contents are from return value of handler method.
+ *
+ * @see LineMessageHandlerSupport#handleReturnValue(Event, Object)
+ */
+@Slf4j
+@Builder
+class ReplyByReturnValueConsumer implements Consumer<Object> {
+    private final LineMessagingClient lineMessagingClient;
+    private final Event originalEvent;
+
+    @Component
+    @AllArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
I recommend to implement the constructor by your hand. Because this class has only 1 field :)